django/db/backends/sqlite3/operations.py

```diff
@@ -77,14 +77,22 @@ def format_for_duration_arithmetic(self, sql):
         """Do nothing since formatting is handled in the custom function."""
         return sql
 
-    def date_trunc_sql(self, lookup_type, field_name):
-        return "django_date_trunc('%s', %s)" % (lookup_type.lower(), field_name)
+    def date_trunc_sql(self, lookup_type, field_name, tzname=None):
```

Comment:
An extra argument to the `date_trunc_sql()` and `time_trunc_sql()` should be documented in [Django 3.2 release notes](https://docs.djangoproject.com/en/dev/releases/3.2/#database-backend-api).

Reply:
In the last commit I updated the docstring for both methods, and I added an entry in the release note